message_definitions/v1.0/uAvionix.xml

```diff
@@ -118,5 +130,17 @@
       <description>Transceiver heartbeat with health report (updated every 10s)</description>
       <field type="uint8_t" name="rfHealth" enum="UAVIONIX_ADSB_RF_HEALTH" display="bitmask">ADS-B transponder messages</field>
     </message>
+    <message id="10004" name="UAVIONIX_ADSB_OUT_CFG_REGISTRATION">
+      <description>Aircraft Registration.</description>
```

Comment:
When is this message sent or received?  Note the other messages in here tell you a bit about that.

Reply:
at the request of uavionix, we're going to remove this. Its actually harder for them to get their stuff FAA certified if the normal pilot "tools" set everything. They want to have a different app set set-once type things which is an app they're making.